docs/pages/protocol/signatures.mdx

```diff
@@ -4,11 +4,61 @@ description: "Learn about account signature types when using XMTP"
 
 # Account signatures with XMTP
 
-Learn about the types of account signatures you may be prompted to provide when using apps built with XMTP.
+Learn about the types of account signatures you might be prompted to provide when using apps built with XMTP.
 
-## Sign to send and receive messages using apps built with XMTP
+## Sign to create an inbox (first-time use)
 
-The first time you use an app built with XMTP (Extensible Message Transport Protocol) to send and receive messages using a blockchain account (account), you’re prompted to provide two signatures using your account keys:
+The first time you use an app built with XMTP, a **Sign this message?** window displays to request that you sign an **XMTP : Authenticate to inbox** message. Specifically, the message requests that you sign a **Create inbox** message with your wallet account. 
+
+Sign the message with your wallet account to consent to creating an XMTP inbox owned by your wallet account.
```

Comment:
Some slight nuance here. It makes sense to simplify or omit details as developers don't need to understand completely, but just explaining here so that you have the full context:

1 - The first usage (or 'sign in') of any installation on V3 (not just the first installation) will always generate a signature request. No matter what we need to sign an installation key ('- Grant messaging access to app'). Additionally, as you mention here, if the user has never used V3 before we also need to create an inbox ('- Create inbox'). However, regardless of which case it is, on V3 we will always [roll all signature requests into a single user-facing signature](https://github.com/xmtp/XIPs/blob/main/XIPs/xip-46-multi-wallet-identity.md#signing-text) with the header 'XMTP : Authenticate to inbox', so devs don't need to worry about exactly what is happening under the hood.

2 - If the app is a dual v2 + v3 app, or if the app is being upgraded from v2 to v3, we ideally don't ask for the wallet signature in (1) given the user has already given a bunch of signatures for v2. What we do is we take the v2 identity key and use that to sign the request in (1) instead of the wallet, so the user (and developer) don't have to see anything. This 'optimization' only happens for the *first* installation to 'convert' to v3, so there is some nuance here. In summary, if you are a dual v2 + v3 app or you are being upgraded from v2 to v3, we will 'sometimes' use your v2 credential to skip the v3 signature.

Sorry if this is muddy, def happy to clarify as it's confusing!

Reply:
This is fantastic - thank you so much to the greatest teacher of complex concepts!